shell/components/form/Error.vue

```diff
@@ -0,0 +1,37 @@
+<script>
+export default {
+  props: {
+    value: {
+      default: null,
+      type:    Object
+    },
+    rules: {
+      default:   () => [],
+      type:      Array,
+      // we only want functions in the rules array
+      validator: rules => rules.every(rule => ['function'].includes(typeof rule))
```

Comment:
> can't access property "$store", this is undefined

Took me a while to dig into, best guess at the moment is that this happened when I converted the component mixin over to TS because it wasn't doing that before. It's a little annoying because I'd like to provide the required message as well as the asterix on the field even when it's marked as required using the old method but it's not really required for this PR and it's probably best to eventually sunset the "required" prop on labeled-form-elements in favor of the newer form validation anyways. So for now I've removed the offending bit of code which doesn't break anything that was already there but also doesn't add in the backwards compatibility for explicit error messages on the field for required inputs.